web/courses.go

```diff
@@ -533,6 +535,34 @@ func (s *AutograderService) setLastApprovedAssignment(submission *pb.Submission,
 	return s.db.UpdateEnrollment(query)
 }
 
+// acceptRepositoryInvites tries to accept repository invitations for the given repository types on behalf of the given user.
+func (s *AutograderService) acceptRepositoryInvites(ctx context.Context, user *pb.User, course *pb.Course) {
+	user, err := s.db.GetUser(user.ID)
+	if err != nil {
+		s.logger.Errorf("Failed to get user %d: %v", user.ID, err)
+		return
+	}
+	userSCM, err := s.getSCM(ctx, user, "github")
+	if err != nil {
+		s.logger.Errorf("Failed to get SCM for user %d: %v", user.ID, err)
+		return
+	}
+	opts := &scm.RepositoryInvitationOptions{
+		Login: user.Login,
+		Owner: course.GetOrganizationPath(),
+	}
+	invites, err := userSCM.GetRepositoryInvites(ctx, opts)
```

Comment:
Suggest that we combine `GetRepositoryInvites` and `AcceptRepositoryInvite` into one method called `AcceptRepositoryInvites` that does both functionalities. I don't think we need the `Get` method for anything else. That way we reduce the methods in the `scm` interface as well.